sds_data_manager/lambda_code/SDSCode/path_helper.py

```diff
@@ -1,43 +1,29 @@
 """Functions to store file pattern, validate and construct upload path
 """
 import re
-from dataclasses import dataclass, field
 from datetime import datetime
 
+VALID_INSTRUMENTS = {
+    "codice",
+    "glows",
+    "hit",
+    "hi-45",
+    "idex",
+    "lo",
+    "mag",
+    "swapi",
+    "swe",
+    "ultra-45",
+}
 
-@dataclass
-class FilenamePatternConfig:
-    """This class stores filename pattern configuration."""
-
-    mission: str = "imap"
-    instruments: list = field(
-        default_factory=lambda: [
-            "codice",
-            "glows",
-            "hit",
-            "hi-45",
-            "idex",
-            "lo",
-            "mag",
-            "swapi",
-            "swe",
-            "ultra-45",
-        ]
-    )
-    data_level: list = field(
-        default_factory=lambda: ["l0", "l1", "l1a", "l1b", "l1c", "l1d", "l2"]
-    )
-    descriptor: str = None
-    startdate: str = "YYYYMMDD"
-    enddate: str = "YYYYMMDD"
-    version: str = r"^v\d{2}-\d{2}$"
-    extensions: list = field(default_factory=lambda: ["pkts", "cdf"])
-
-
-class FilenameParser:
+VALID_DATALEVELS = {"l0", "l1", "l1a", "l1b", "l1c", "l1d", "l2"}
+
+VALID_FILE_EXTENSION = ["pkts", "cdf"]
```

Comment:
For consistency with the sets above
```suggestion
VALID_FILE_EXTENSION = {"pkts", "cdf"}
```